### PR TITLE
Shcherbak Search-bar

### PR DESCRIPTION
**Pull request by itpsychologist**
- [DEMO LINK](https://itpsychologist.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://itpsychologist.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
```
Comment (mykhalenych) [inline src/index.html:14]: Screenshot 2023-01-18 at 18 26 32

check all elements

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+@font-face {
+  font-family: "Avenir-bold";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: 900;
+}
+
+/* html,
```
Comment (mykhalenych) [inline src/style.css:14]: delete comments

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  width: 100%;
+  height: 42px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+}
+
+/* .search-icon {
```
Comment (mykhalenych) [inline src/style.css:50]: the same, check all file

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  -webkit-appearance: none;
+}
+
+.big-search {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
```
Comment (mykhalenych) [inline src/style.css:30]: move to * selector

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+}
+
+.small-search {
+  display: flex;
```
Comment (mykhalenych) [inline src/style.css:38]: you duplicate to much styles, create common class for two inputs and move common styles to this class

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+.small-search > .search-label {
+  display: flex;
+  align-items: center;
+  margin-left: 13px;
+}
+
+.search-text {
+  border: none;
+  outline: none;
+  font-family: "Avenir-bold", sans-serif;
```
Comment (mykhalenych) [inline src/style.css:75]: move font-family to html selector

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+.small-search > .search-label > .search-text::placeholder {
+  /* font-family: inherit; */
+  font-size: 14px;
+  font-weight: 400px;
+  color: #3d4e61;
+}
+
+.big-search:hover,
+.small-search:hover {
+  border: 1px solid #e1e7ed;
```
Comment (mykhalenych) [inline src/style.css:118]: when you use color more than 1 time create variable and use it instead of e1e7ed

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+.search-text:active {
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.small-search > .search-label > .search-text {
+  border: none;
+  outline: none;
+  font-family: "Avenir-Heavy", sans-serif;
```
Comment (mykhalenych) [inline src/style.css:95]: and delete here

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+  font-style: normal;
+  font-weight: 900;
+  font-size: 14px;
+  line-height: 19px;
+  margin-left: 9px;
+}
+
+.search-text::placeholder {
+  font-family: "Avenir", sans-serif;
```
Comment (mykhalenych) [inline src/style.css:104]: and here as well

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+  margin-left: 17px;
+  padding: 0;
+}
+
+.small-search > .search-text {
+  margin-left: 9px;
+  padding: 0;
+}
+
+.search-text:active {
```
Comment (mykhalenych) [inline src/style.css:88]: general: try don't use such construction, use only class

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Focus on inputs different from design

File: src/index.html [hunk 11]
```diff
@@ -10,7 +10,36 @@
...
+  <body class="main">
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    class="search-form form-big"
+    data-qa="big"
+    >
+      <label class="search-label label-big">
+        <input
+        type="search"
```
Comment (mykhalenych) [inline src/index.html:22]:
```suggestion
        <input
           type="search"
```

File: src/index.html [hunk 12]
```diff
@@ -10,7 +10,36 @@
...
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        class="search-text text-big"
+        >
+      </label>
+    </form>
+    <form
```
Comment (mykhalenych) [inline src/index.html:29]: check all tags
Screenshot 2023-01-22 at 14 37 03

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+
+.label-big::after {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.label-small::after {
+
```
Comment (mykhalenych) [inline src/style.css:68]: delete space

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Check comments, also focus doesn't work, check design

File: src/style.css [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-text {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
```
Comment (IvanFesenko) [inline src/style.css:78]: Use `padding` instead `height`

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
2023-02-02_15h12_06
2023-02-02_15h12_10

Styles on focus didn't much design

File: src/style.css (hunk 14 above)
Comment (itpsychologist) [reply src/style.css:78]: I don't understand, what do you meen? 
In my project focus don't have design or at Figma?

**Pull request closed without merging**